src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/CalendarDatePickerDateChangedEventArgs.cs

```diff
@@ -16,8 +16,8 @@ public  partial class CalendarDatePickerDateChangedEventArgs
 				throw new global::System.NotImplementedException("The member DateTimeOffset? CalendarDatePickerDateChangedEventArgs.NewDate is not implemented in Uno.");
 			}
 		}
-		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#endif
+#if false || false || false || false || false
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Correct one of the identical expressions on both sides of operator '||'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826528)